bridge/config/config.go

```diff
@@ -185,7 +185,7 @@ func NewConfig(cfgfile string) *Config {
 	viper.SetConfigFile(cfgfile)
 	viper.SetEnvPrefix("matterbridge")
 	viper.AddConfigPath(".")
-	viper.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
+	viper.SetEnvKeyReplacer(strings.NewReplacer(".", "_", "-", ""))
```

Comment:
I think it's better to replace dash to underscore 
`viper.SetEnvKeyReplacer(strings.NewReplacer(".", "_", "-", "_"))`

Reply:
I guess I liked that the underscores mapped to the keyspace, but I can change it